src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/EventTypeAT.java

```diff
@@ -0,0 +1,56 @@
+package de.zalando.aruha.nakadi.webservice;
+
+import com.jayway.restassured.response.Response;
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.EventTypeSchema;
+import org.apache.http.HttpStatus;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.jayway.restassured.RestAssured.given;
+import static java.text.MessageFormat.format;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class EventTypeAT extends BaseAT {
+    @Test
+    public void whenPOSTValidEventTypeThenOk() {
+        Map body = buildEventTypeBody("event-name");
+
+        final Response response = given().body(body).post("http://localhost:8080/event_types");
+
+        response.then().body(equalTo(""));
+        response.then().statusCode(HttpStatus.SC_CREATED);
+    }
+
+    @Test
+    public void whenPOSTInvalidEventTypeThenUnprocessableEntity() {
+        Map body = buildEventTypeBody(null);
+
+        final Response response = given().body(body).post("http://localhost:8080/event_types");
+
+        response.then().body(equalTo(
+                "{\"detail\":\"#/event-type/name: expected type: String, found: Null\",\"title\":\"Invalid EventType object\",\"status\":422,\"type\":\"https://httpstatuses.com/422\"}"));
```

Comment:
You can use a JsonMatcher as well: :)
https://www.google.de/webhp?ie=UTF-8#q=hamcrest%20json%20matcher